tests/math/test_functions.py

```diff
@@ -674,6 +691,38 @@ def test_autograd(self):
         t = np.array([1.0, 2.0], requires_grad=False)
         assert not fn.requires_grad(t)
 
+    def test_autograd_backwards(self):
+        """jax.DeviceArrays will always return True"""

```

Comment:
maybe a bit more info from the docstring would help? Like "will always return True in a backwards pass"(?)

Reply:
wait, this is a typo right? the test is not about jax :)

Reply:
Oops 🤦 
```suggestion
        """Autograd trainability corresponds to the requires_grad attribute during the backwards pass."""
```